src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -1296,6 +1317,10 @@ namespace OpenMS
           peptide_hit_.setMetaValue("num_matched_peptides", value);
         }
       }
+
+      double pepmassdiff = attributeAsDouble_(attributes, "massdiff");
```

Comment:
Is this mandatory or can it be optional in pepXML?

Reply:
required

```
<xs:attribute name="massdiff" type="xs:string" use="required">
<xs:annotation>
<xs:documentation>Mass(precursor ion) - Mass(peptide)</xs:documentation>
</xs:annotation>
</xs:attribute>
```